### PR TITLE
Adding second map from imported data

### PR DESCRIPTION
**Pull request by lqueipom**

### CODE REVIEW

**Comment by LukeGMTravis** [comment]
Starting review

File: portfolio/src/main/webapp/WEB-INF/favorites.csv [hunk 1]
```diff
@@ -0,0 +1,5 @@
+10.655559,-71.594271
```
Comment (LukeGMTravis) [inline portfolio/src/main/webapp/WEB-INF/favorites.csv:1]: Nit: space

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -178,4 +178,24 @@ function visualizeMap() {
   map.setMapTypeId('styles_map');
   homeInfoWindow.open(map, markerHome);
   lakeInfoWindow.open(map, markerLake);
+
+  favoritePlaces();
```
Comment (LukeGMTravis) [inline portfolio/src/main/webapp/script.js:182]: A more descriptive function name like `loadFavoritePlaces` would be easier to understand at a glance

File: portfolio/src/main/webapp/style.css [hunk 3]
```diff
@@ -1,9 +1,23 @@
...
 
 #greeting-container {
   margin-top: 20px;
 }
+
+#map, #mapTwo{
+  height: 300px;
+  text-align: center;
+  width: 300px;
+  margin-left: auto;
```
Comment (LukeGMTravis) [inline portfolio/src/main/webapp/style.css:16]: Nit: alphabetical order

File: portfolio/src/main/webapp/style.css [hunk 4]
```diff
@@ -1,9 +1,23 @@
...
   margin-right: auto;
+  text-align: center;
   width: 650px;
 }
 
 #greeting-container {
   margin-top: 20px;
 }
+
+#map, #mapTwo{
```
Comment (LukeGMTravis) [inline portfolio/src/main/webapp/style.css:12]: Nit: space

Comment (lqueipom) [reply portfolio/src/main/webapp/style.css]: Thank you for the comments, implemented the changes!

**Pull request merged**